scripts/lint.sh

```diff
@@ -66,4 +66,10 @@ for test in $TESTS; do
   run "${test}" "${TARGET}"
 done
 
+if grep -r "github.com/ava-labs/hypersdk/internal/" ./examples; then
+  echo "Error: example VMs should not use internal packages"
+  exit 1
+fi
+
```

Comment:
Could you add this as a function included in `$TESTS` so that it matches the style of the rest of the linters?

Reply:
+1, we should be able to update our `golangci.yml`'s `depguard` config to blacklist this package from the `examples` dir ([ref](https://github.com/OpenPeeDeeP/depguard))

Reply:
I've added a rule to depcheck, but I'm having trouble testing how it works. It's not working for me at all. Will try to come back to it later today